### PR TITLE
Added Prerequisites for openSUSE (Tumbleweed) and Rust installation

### PR DESCRIPTION
**Pull request by JmbFountain**
Hello,

Here is the updated PR, I'm reissuing it as I am not as familiar with how Github handles PRs. 
I added Dependency installation instructions fro openSUSE and also the installation of Rust (and cargo-make)

Changes/Edits are welcome

### CODE REVIEW

File: essential-documentation/contribute-to-appflowy/software-contributions/environment-setup/building-on-linux.md [hunk 1]
```diff
@@ -42,8 +43,28 @@ sudo apt-get install curl build-essential libsqlite3-dev libssl-dev clang cmake
...
+{%tab title="openSUSE Tumbleweed" %}
+```bash
+sudo zypper install -t pattern devel_basis
+sudo zypper install libopenssl-devel sqlite3-devel clang cmake ninja pkgconf-pkg-config pkgconf libgtk-3-0 curl zip unzip gtk3-devel
+```
+{% endtab %}
+
 {% endtabs %}
 
+* Install Rust according to [https://www.rust-lang.org/tools/install](https://www.rust-lang.org/tools/install).
```
Comment (MikeWallaceDev) [inline essential-documentation/contribute-to-appflowy/software-contributions/environment-setup/building-on-linux.md:56]: Rust is installed via the install script, please remove this section.

File: essential-documentation/contribute-to-appflowy/software-contributions/environment-setup/building-on-linux.md [hunk 2]
```diff
@@ -42,8 +43,28 @@ sudo apt-get install curl build-essential libsqlite3-dev libssl-dev clang cmake
 yay -S curl base-devel sqlite openssl clang cmake ninja pkg-config gtk3 unzip
 ```
 {% endtab %}
+
+{%tab title="openSUSE Tumbleweed" %}
+```bash
+sudo zypper install -t pattern devel_basis
+sudo zypper install libopenssl-devel sqlite3-devel clang cmake ninja pkgconf-pkg-config pkgconf libgtk-3-0 curl zip unzip gtk3-devel
```
Comment (MikeWallaceDev) [inline essential-documentation/contribute-to-appflowy/software-contributions/environment-setup/building-on-linux.md:50]: is it necessary to install libopenssl? (sorry if that sounds dumb, but I already had it installed on my system...)

**Comment by MikeWallaceDev** [comment]
Thank you for your contribution! It's simple to make changes to this PR. Just continue working in the branch that you were using, commit your changes, then push them to your github account (like a normal push) and your changes will automatically update this PR.

**Comment by annieappflowy** [comment]
Hi @JmbFountain , are you still working on this? If not, I will close it.

**Comment by JmbFountain** [comment]
You are fine to close it, as I don't really have the time to work on this anymore.

**Pull request closed without merging**